### PR TITLE
Fix http-add-on operator resources

### PR DESCRIPTION
**Pull request by aballman**
The http-add-on helm chart has been updated so the `operator` component's deployment uses the `resources` specified in the helm chart values, and allows them to be overwritten.

### Checklist

- [x] I have verified that my change is according to the [deprecations & breaking changes policy](https://github.com/kedacore/governance/blob/main/DEPRECATIONS.md)
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/charts/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))
- ~README is updated with new configuration values *(if applicable)* [learn more](https://github.com/kedacore/charts/blob/main/CONTRIBUTING.md#documentation)~
- ~A PR is opened to update KEDA core ([repo](https://github.com/kedacore/keda)) *(if applicable, ie. when deployment manifests are modified)*~

Fixes #
https://github.com/kedacore/charts/issues/565

### CODE REVIEW

File: http-add-on/templates/operator/deployment.yaml
Comment (JorTurFer) [inline http-add-on/templates/operator/deployment.yaml:37]: These configs are for the RbacProxy, not for the operator itself. The operator is already reading the config from values. I agree with adding parameters for configuring these options, but they have to be split from operator as this is just for metrics

Comment (aballman) [reply http-add-on/templates/operator/deployment.yaml:37]: Thanks for this review here as well! The requested updates have been made. This is failing the CI process though the same as the other PR I have open:

https://github.com/kedacore/charts/pull/568#discussion_r1408158455

Comment (JorTurFer) [reply http-add-on/templates/operator/deployment.yaml:37]: yeah, let's wait until we merge [the fix for the CI](https://github.com/kedacore/charts/pull/582)

**Comment by JorTurFer** [comment]
Could you rebase your branch?

**Comment by aballman** [comment]
> Could you rebase your branch?

Done!

**Review by JorTurFer — APPROVED** [review]
LGTM!
Thanks for the improvement ❤️

**Pull request merged**
